config/defaults.js

```diff
@@ -217,5 +217,10 @@ module.exports = {
     expirationAfterSession: 15 * 60,
     //base path of the socket artifact
     artifactName: 'private/docker-worker/'
+  },
+
+  retry: {
```

Comment:
I think this should be named to describe what the retry is for. (I hate naming so I don't have a good name here :) )